events/events.json

```
@@ -1,5 +1,13 @@
 {
   "events": [
+    {
+      "location": "Nagoya, Japan",
+      "writtenDate": "March 27th - 28th, 2026",
+      "startDate": "2026-03-27",
+      "endDate": "2026-03-28",
+      "imageUrl": "/images/nagoya/rails_girls_nagoya_6th_logo.png",
```

@Ocha-Misaki 

The image is not displayed because the file does not exist at the given path.
Also, the file name is nagoya_6th, but this is the 7th event.

If the default image is acceptable, please commit suggest.

```suggestion
      "imageUrl": "/images/railsgirls-sq.png",
```